client/verta/verta/deployment/__init__.py

```diff
@@ -4,4 +4,5 @@
     DeployedModel,
     prediction_input_unpack,
     prediction_io_cleanup,
+    prediction_input_df
```

Comment:
Trailing commas preferred!
```suggestion
    prediction_input_df,
```

Reply:
fixed.